src/sidebar/util/retry.js

```diff
@@ -1,29 +1,41 @@
 import retry from 'retry';
 
+/**
+ * Options passed that control how the operation is retried.
+ *
+ * See https://github.com/tim-kos/node-retry#retrytimeoutsoptions
+ *
+ * @typedef RetryOptions
+ * @prop {number} minTimeout
+ */
```

Comment:
This type is incomplete. I've only included the properties that we've actually found a use for so far.